### ISSUE TITLE
Network not working with rectangular input?

### ISSUE THREAD

**Issue by beinanwang** [pjreddie/darknet#445]
I have the network below. It takes a picture of 90 x 30 and outputs 2 probablities (dog vs cat). The problem is after a few epochs the loss stops to decrease and when I use it to classify it predicts dog for all images. This issue seems to only happen when the input is rectangular.

However, if I implement the same network with Keras in Python it will work perfectly fine. Any thoughts?

[net]
batch=128
subdivisions=1
width=30
height=90
channels=3
momentum=0.9
decay=0.0005

learning_rate=0.01
policy=poly
power=4
max_batches=800000

[convolutional]
batch_normalize=1
filters=40
size=5
stride=1
pad=0
activation=relu

[convolutional]
batch_normalize=1
filters=60
size=5
stride=1
pad=0
activation=relu

[convolutional]
batch_normalize=1
filters=80
size=3
stride=1
pad=0
activation=relu

[connected]
output=500
activation=relu

[connected]
output=2
activation=linear

[softmax]
groups=1

[cost]
type=sse
